src/ndarray/ndarray.cc

```diff
@@ -733,6 +739,21 @@ void NDArray::UpdateMKLDNNMemDesc() {
 }
 #endif
 
+#if MXNET_USE_NGRAPH == 1
+std::shared_ptr<ngraph::runtime::TensorView> &NDArray::create_tensor_view() {
```

Comment:
why do you return a reference of a shared pointer?

Reply:
mainly for perf. reasons, but RVO may optimize this away will change to return by value.